Makefile

```diff
@@ -97,7 +103,11 @@ $(libff_out): $(PLUGIN_SUBMODULE)/deps/libff/CMakeLists.txt
 	cd $(PLUGIN_SUBMODULE)/deps/libff/build                                                                     \
 	    && cmake .. -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=$(INSTALL_LIB)/libff $(LIBFF_CMAKE_FLAGS) \
 	    && make -s -j4                                                                                          \
-	    && make install DESTDIR=$(CURDIR)/$(BUILD_DIR)
+	    && make install DESTDIR=$(BUILD_DIR)
```

Comment:
This change is not correct I don't think. Pretty sure it needs to be `$(CURDIR)/$(BUILD_DIR)`, or it will be installed in the submodule build directory, not in this ones build dircetory.

Reply:
Did you see the change I made to `BUILD_DIR`? I needed to make it an absolute path, I think it was for the CMakeLists.txt file.